base/libgit2/repository.jl

```diff
@@ -313,6 +313,24 @@ function checkout_head(repo::GitRepo; options::CheckoutOptions = CheckoutOptions
                  repo.ptr, Ref(options))
 end
 
+"""
+    LibGit2.cherrypick(repo::GitRepo, commit::GitCommit; options::CherrypickOptions = CherrypickOptions())
+
+Cherrypick the commit `commit` and apply the changes in it to the current state of `repo`.
+The keyword argument `options` sets checkout and merge options for the cherrypick. If `null`,
```

Comment:
`C_NULL` but in fact we just use the default values anyway so I should probably delete that (it's leftover from when that kwarg was a `Nullable`, very nasty).